### ISSUE TITLE
Levin - CPU Pick up rules

### ISSUE THREAD

**Issue by i-rabot** [kbower/tickettest1#2694]
_Imported from trac ticket 2694.
Created by **yheffington**
Opened in trac: Mar 13, 2014, 2:58:39 PM
Last modified in trac: Mar 26, 2014, 12:47:56 PM_
***
_by yheffington:_
**CPU Pick up** - not fully defined
 - Print PU copy on demand only once.  Printed when customer arrives to pick up and order is paid in full.
 - Permission for printing after initial print
 - ?

**Comment by i-rabot** [comment]
_**sjohnson** trac status change on Mar 13, 2014, 2:58:55 PM:_ **waiting**

**Comment by i-rabot** [comment]
_**kbower** trac comment on Mar 24, 2014, 4:43:02 PM:_

Customer Pickup (not take with) - Currently the pick up copy is printed at the time of sale when picking up at the store or in the pick up office at the warehouse prior to going to the dock.  Information is collected on the copy (lic #, etc) and items are circled as loaded.  Restrict printing of the pick up slip copy in Loft unless the order is Open and Paid in Full.  A permission will be added to allow printing without these restrictions.  The office will need to Close the order to finalize it.

**Comment by i-rabot** [comment]
_**kbower** trac comment on Mar 26, 2014, 12:47:56 PM:_

(Changeset [[5221]](../commit/37e188d531b9367ae85f7a9d84c9c47573f96b4e))
* closes #2694, Levin CPU Pick-up slip printing #lev
   * Restrict printing of the pick up slip in Loft unless the order is Open and paid in full.
   * Only allow pickup slip to be printed once.
   * Permission 'levpickupslip' allows re-printing or printing with balance.
 * split tickets with $0.01 balance due to tax rounding artifact are considered 'fully funded' #lev
 * allow scheduling delivery for open balance ticket when a sibling ticket contains 'HOLDCHECK', 'BILLED', 'GIFTCERT', or 'STATEFARM' (closes #2679)
 * default new ticket delivery date to 'Schedule ASAP' 12/5/2049 #lev

**Comment by i-rabot** [comment]
_**kbower** trac status change on Mar 26, 2014, 12:47:56 PM:_ **closed**
